completions/dolphin

```diff
@@ -0,0 +1 @@
+complete -W '--help --help-qt --help-kde --help-all --author -v --version --license' dolphin
```

Comment:
This also overwrites the default completion of bash-completion. We would like to handle basic completions provided by `_comp_initialize` also for this command. By default, these are provided as the default completion `_minimal` of `bash-completion` but will be lost by this setting.